cpp/include/cudf/utilities/traits.hpp

```diff
@@ -397,7 +397,10 @@ template <typename T>
 constexpr inline bool is_fixed_point()
 {
   return std::is_same_v<numeric::decimal32, T> || std::is_same_v<numeric::decimal64, T> ||
-         std::is_same_v<numeric::decimal128, T>;
+         std::is_same_v<numeric::decimal128, T> ||
+         std::is_same_v<numeric::fixed_point<int32_t, numeric::Radix::BASE_2>, T> ||
+         std::is_same_v<numeric::fixed_point<int64_t, numeric::Radix::BASE_2>, T> ||
+         std::is_same_v<numeric::fixed_point<__int128_t, numeric::Radix::BASE_2>, T>;
```

Comment:
This looks concerning. I don't believe libcudf supports these types.
Why were these added?

Reply:
Ok, Is this change necessary though? Will something break without it?
We may need to reevaluate this.
My fear is that this function is used in type-dispatching and similar type-check logic and I don't think any of our code paths are expecting base-2 presently.

Reply:
E.g. the SimpleBinaryFPConstruction test

Reply:
Right, I'm thinking we just consider removing that test.

Reply:
This is needed because I'm using is_fixed_point with enable_if on these convert functions to make sure that the input types are valid.  I could instead create is_binary_fixed_point() for that. 

Though of course if this is an issue, the real problem is that is_fixed_point() is a misnomer and should be something like is_decimal_fixed_point(), but that would require changing tons of files in the code base.  Or if other bases will never be supported, I could nuke those entirely. 

I'm pretty sure these fixed_point tests are the only place where base-2 is used.

Reply:
That's ok. I just wanted to consider it a bit. 
I don't want to create `is_binary/is_decimal` functions until it becomes necessary.